enhancements/update/eus-upgrades-mvp.md

```diff
@@ -0,0 +1,368 @@
+---
+title: eus-upgrades-mvp
+authors:
+  - "@sdodson"
+reviewers:
+  - @darkmuggle
+  - @soltysh
+  - @sttts
+  - @deads2k
+  - @ecordell
+  - @dgoodwin
+approvers:
+  - "@eparis"
+  - "@derekwaynecarr"
+  - "@crawford"
+  - "@pweil-"
+creation-date: 2020-01-12
+last-updated: 2020-01-12
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# EUS Upgrades MVP
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement outlines a set of cross platform improvements meant to ensure
+the safety of multiple back-to-back minor version upgrades associated with EUS
+to EUS upgrades. Additionally, we outline improvements applicable to all
+upgrades meant to reduce the duration of upgrades and workload disruption.
+
+All of the work detailed herein is prerequisite to enabling upgrades which skip
+reboots. However where non-overlapping resources exist we can pursue reboot
+removal in parallel such as the Node team validating upstream's stated Kubelet
+version skew policies.
+
+## Motivation
+
+The introduction of EUS creates a subset of clusters which we expect will run
+4.6 for a year or more then upgrade rapidly, though serially, from 4.6 to 4.10.
+This rapid upgrade introduces the risk that those clusters may upgrade faster
+than is safe due to constraints imposed by OpenShift, the upstream components of
+OpenShift, or deployed workloads.
+
+This also creates a scenario where admins wish to reduce both the duration and
+the disruption to workload associated with the upgrade.
+
+### Goals
+
+- We will inform admins of incompatibilities between their current usage and the
+final upgrade target
+- We will provide disconnected admins tools that help them plan their entire
+upgrade path from 4.6 to 4.10 so that they may mirror all content necessary
+- We will provide guidance on expected upgrade duration scaled by node count exclusive
+of the variability associated with workload rescheduling (ie: everything except
+Worker MCP rollout)
```

Comment:
nit: expand MCP to MachineConfigPool?  I was scratching my head for a minute until that clicked.